bundles/EcommerceFrameworkBundle/IndexService/ProductList/ElasticSearch/AbstractElasticSearch.php

```diff
@@ -1253,14 +1255,31 @@ protected function sendRequest(array $params)
                 $params['body']['size'] = $this->getLimit();
             }
 
+            $eventDispatcher = \Pimcore::getEventDispatcher();
+            $preSendRequestEvent = new PreSendRequestEvent($params);
+            $eventDispatcher->dispatch(IndexServiceEvents::PRE_SEND_REQUST_ELASTIC_SEARCH, $preSendRequestEvent);
+
+            if ($preSendRequestEvent->getStopRequest()) {
+                return $result;
```

Comment:
this will always be an empty result? is that really wanted? 
should we not add the option to modify the result in event listener?